src/Widgets/ExportWidget.vala

```diff
@@ -108,33 +105,32 @@ public class Akira.Widgets.ExportWidget : Gtk.Grid {
         yield get_image_buffer_size ();
 
         double bytes = (double) imagedata.length;
-        double full_bytes = imagedata.length > MB
-            ? bytes / MB : bytes / KB;
+        double full_bytes = imagedata.length > MB ? bytes / MB : bytes / KB;
         var size = imagedata.length > MB
             ? ("%0.1fMB").printf (full_bytes)
             : ("%0.1fKB").printf (full_bytes);
 
-        info.label = _("%i × %i px · %s").printf (
-            model.pixbuf.width,
-            model.pixbuf.height,
-            size);
+        info.label = _("%i × %i px · %s").printf (model.pixbuf.width, model.pixbuf.height, size);
     }
 
     private async void get_image_buffer_size () {
+        unowned var image = model.pixbuf;
         info.label = _("Fetching image size…");
         SourceFunc callback = get_image_buffer_size.callback;
 
         new Thread<void*> (null, () => {
             try {
                 if (settings.export_format == "png") {
```

Comment:
settings.export_format should really be an enum :slightly_smiling_face:

Reply:
Note that you can keep the enum<->string binding (especially for `bind_property`) with [GLib.Value.register_transform_func](https://valadoc.org/gobject-2.0/GLib.Value.register_transform_func.html)

Reply:
Ah, good suggestion.
I'll take care of this in a dedicated follow up when I add more export formats.